.circleci/config.yml

```diff
@@ -0,0 +1,145 @@
+version: 2
```

Comment:
Could you please update the `circle_tests.py` script which allows to run all commands from the circle.yml file and emulate circle CI test run?

Reply:
Now works across both versions!